ChangeLog.md

```diff
@@ -29,6 +29,11 @@ repository history](https://github.com/ddclient/ddclient/commits/master).
     `use=ip,ip=<ipv4-address>` to fail.
   * "true" is now accepted as a boolean value.
 
+### Removed features
+
+  * Removed dtdns provider
+  * Removed Hammernode provider
```

Comment:
I doubt anyone is using these, so they don't need to be highlighted in their own section. Perhaps move these to the compatibility changes section.

Also maybe word them as "Removed support for defunct ____ service"